NoSQL/README.md

```diff
@@ -4,79 +4,129 @@ Sample Docker build files to facilitate installation and environment setup for
+docker run -d --name=kvlite --env KV_PROXY_PORT=8080 -p 8080:8080 oracle/nosql:20.3
 ```
 
-In a second shell, run a second Docker container to ping the kvlite store
+In a second shell, run a second container to ping the kvlite store
 instance:
 
 ```shell
-docker run --rm -ti --link kvlite:store oracle/nosql:19.5-ce \
+docker run --rm -ti --link kvlite:store oracle/nosql:20.3 \
   java -jar lib/kvstore.jar ping -host store -port 5000
 ```
 
 Note the required use of `--link` for proper hostname check (actual KVLite
 container is named `kvlite`; alias is `store`).
 
 You can also use the Oracle NoSQL Command Line Interface (CLI). Start the
-following container (keep container `kvlite` running):
+following container:
 
 ```shell
-$ docker run --rm -ti --link kvlite:store oracle/nosql:19.5-ce \
+
+$ docker run --rm -ti --link kvlite:store oracle/nosql:20.3  java -Xmx64m -Xms64m -jar lib/kvstore.jar version
+
+20.3.19 2021-09-29 04:04:01 UTC  Build id: b8acf274b357 Edition: Community
+
+$ docker run --rm -ti --link kvlite:store oracle/nosql:20.3 \
   java -jar lib/kvstore.jar runadmin -host store -port 5000 -store kvstore
 
   kv-> ping
-  Pinging components of store kvstore based upon topology sequence #14
-  10 partitions and 1 storage nodes
-  Time: 2017-02-28 15:37:41 UTC   Version: 12.1.4.3.11
-  Shard Status: healthy:1 writable-degraded:0 read-only:0 offline:0
-  Admin Status: healthy
-  Zone [name=KVLite id=zn1 type=PRIMARY allowArbiters=false]   RN Status: online:1 offline:0
-  Storage Node [sn1] on 659dbf4fba07:5000
-  Zone: [name=KVLite id=zn1 type=PRIMARY allowArbiters=false]
-  Status: RUNNING  Ver: 12cR1.4.3.11 2017-02-17 06:52:09 UTC  Build id: 0e3ebe7568a0
-  Admin [admin1]     Status: RUNNING,MASTER
-  Rep Node [rg1-rn1] Status: RUNNING,MASTER sequenceNumber:49 haPort:5006
+   Pinging components of store kvstore based upon topology sequence #14
+   10 partitions and 1 storage nodes
+   Time: 2021-12-20 12:56:33 UTC   Version: 20.3.19
+   Shard Status: healthy:1 writable-degraded:0 read-only:0 offline:0 total:1
+   Admin Status: healthy
+   Zone [name=KVLite id=zn1 type=PRIMARY allowArbiters=false masterAffinity=false]   RN Status: online:1 read-only:0 offline:0
+   Storage Node [sn1] on dcbd8ff4f07c:5000    Zone: [name=KVLite id=zn1 type=PRIMARY allowArbiters=false masterAffinity=false]    Status: RUNNING   Ver: 20.3.19 2021-09-29 04:04:01 UTC  Build id: b8acf274b357 Edition: Community
+        Admin [admin1]          Status: RUNNING,MASTER
+        Rep Node [rg1-rn1]      Status: RUNNING,MASTER sequenceNumber:50 haPort:5003 available storage size:1023 MB
+
 
   kv-> put kv -key /SomeKey -value SomeValue
   Operation successful, record inserted.
   kv-> get kv -key /SomeKey
   SomeValue
-  kv->
+  kv-> exit
 ```
 
+You can also use the Oracle SQL Shell Command Line Interface (CLI). Start the
+following container:
+
+```shell
+$ docker run --rm -ti --link kvlite:store oracle/nosql:20.3 \
```

Comment:
Yes, it is the expected behavior. I am working to allow users to execute those commands from the host or another host. Instead of running another container just to connect to the NoSQL server. Those commands are running the NoSQL utilities. This is why the -ti and -rm.